--- pixieplus-0.5.4/app/imageheaders.cpp.orig	2003-02-09 04:56:50.000000000 +0000
+++ pixieplus-0.5.4/app/imageheaders.cpp	2004-04-05 09:53:55.000000000 +0000
@@ -90,7 +90,7 @@
         if(TIFFSetDirectory(t, thumbDir)){
             TIFFGetField(t, TIFFTAG_IMAGELENGTH, &height);
             img.create(minWidth, height, 32);
-            if(!TIFFReadRGBAImage(t, minWidth, height, (unsigned long*)
+            if(!TIFFReadRGBAImage(t, minWidth, height, (uint32*)
                                   img.bits(), 0))
                 img.reset();
             else{
